mod.sp

```diff
@@ -7,6 +7,43 @@ locals {
   }
 }
 
+variable "common_dimensions" {
+  type        = list(string)
+  description = "A list of common dimensions to add to each control."
+  # Define which common dimensions should be added to each control.
+  # - connection_name (_ctx ->> 'connection_name')
+  # - region
+  default = ["region"]
+}
+
+variable "tag_dimensions" {
+  type        = list(string)
+  description = "A list of tags to add as dimensions to each control."
+  default     = []
```

Comment:
```suggestion
  # A list of tag names to include as dimensions for resources that support
  # tags (e.g.  "Department", "Environment"). Default to empty since tag names are
  # a personal choice.
  default     = []
```